packages/happy-dom/test/nodes/html-option-element/HTMLOptionsCollection.test.ts

```diff
@@ -134,16 +128,21 @@ describe('HTMLOptionsCollection', () => {
 			const select = <HTMLSelectElement>document.createElement('select');
 			const option = document.createElement('option');
 			const option2 = document.createElement('option');
+
+			expect(select.options.selectedIndex).toBe(-1);
+
 			select.appendChild(option);
 			select.appendChild(option2);
-			document.body.appendChild(select);
-			expect(select.options.selectedIndex).toBe(-1);
+
+			expect(select.options.selectedIndex).toBe(0);
```

Comment:
This has me confused: the spec says it should be -1 in this situation ([ref](https://html.spec.whatwg.org/multipage/form-elements.html#dom-select-selectedindex)), but the browser says 0 😖. Did you find a place in the spec indicating it should actually be 0, or are you changing this purely based on observations of how browsers actually act?

Reply:
I found it by testing the behavior in the browser, but there is a spec for it.

Spec:
https://html.spec.whatwg.org/multipage/form-elements.html#selectedness-setting-algorithm

Reply:
Ahh, thanks a lot! So it looks like -1 is only a valid initial value if every option is disabled.

Reply:
Yes it seems like that. I will try to fix it so that it works according to spec now. Wish me luck :sweat_smile: